### PR TITLE
lint: enable wsl check

### PR DESCRIPTION
**Pull request by bart0sh**
Fixes: #392

### CODE REVIEW

**Comment by bart0sh** [comment]
@mythi @rojkov does it make sense to continue with this? I'm around a halfway or a bit further.

File: cmd/fpga_tool/fpga_tool.go [hunk 1]
```diff
@@ -34,9 +34,13 @@ const (
 
 func main() {
 	var err error
+
 	var bitstream, device string
+
 	var dryRun, force, quiet bool
+
 	var port uint
+
```
Comment (rojkov) [inline cmd/fpga_tool/fpga_tool.go:43]: I wonder why the check is so aggressive here. Probably the expected format is something like
```go
var (
        err error
        bitstream, device string
        dryRun, force, quiet bool
        port uint
)
```

**Review by rojkov — COMMENTED** [review]
Wow, that's huge!
imho this looks more readable now, but those var's on separate lines don't look eyes candy.

File: cmd/fpga_tool/fpga_tool.go (hunk 1 above)
Comment (bart0sh) [reply cmd/fpga_tool/fpga_tool.go:43]: Thanks for the suggestion. Will try that.

**Comment by bart0sh** [comment]
@rojkov @mythi only one is left:
```
cmd/operator/main.go:56:1: block should not end with a whitespace (or comment) (wsl)
```

Any suggestions how to fix it?

**Review by rojkov — DISMISSED** [review]
Thank you!

**Comment by bart0sh** [comment]
@mythi can you review this PR? It's better to get it merged soon as any other merged PR may require rebase of this one due to its size.

**Review by mythi — DISMISSED** [review]

**Comment by mythi** [comment]
@hj-johannes-lee pls check and approve

**Review by mythi — APPROVED** [review]

**Review by rojkov — APPROVED** [review]

**Pull request merged**